bitnami/etcd/templates/statefulset.yaml

```diff
@@ -157,29 +157,35 @@ spec:
               value: "true"
             {{- else if .Values.auth.client.secureTransport }}
             - name: ETCD_CERT_FILE
-              value: "/opt/bitnami/etcd/certs/client/cert.pem"
+              value: "/opt/bitnami/etcd/certs/client/{{ .Values.auth.client.certFilename }}"
             - name: ETCD_KEY_FILE
-              value: "/opt/bitnami/etcd/certs/client/key.pem"
+              value: "/opt/bitnami/etcd/certs/client/{{ .Values.auth.client.certKeyFilename }}"
```

Comment:
You will also need to change the `_helpers.tpl`, the `NOTES.txt` and the `cronjob.yaml` files.
The same applies to the `cert.pem` file too.

Reply:
`_helper.tpl` seems to be for the autoTLS option. I'm not sure it makes sense to override there. Will add to the other sections you mention